### DISCUSSION TITLE
Hardware recommendations? Ryzen 9 5950x? RTX 3060?

### DISCUSSION THREAD

**Discussion by mschira** [neurodesk/neurodesk.github.io#103] [General]
Dear Neurodesk community and developers
I am currently looking to purchase a new workstation for MRI data analysis.
We are pretty power hungry,

We do more and more Nvidia GPU intensive stuff, too, and getting time on massive is getting harder and harder.
So I am eying an AMD Ryzen 9 5950x with an RTX 3060 and 128GB of RAM.
Are there any issues with this for running Neurodesk containers?
I feel Intel CPUs aren't really competitive atm.

**Comment by stebo85** [comment]
Dear @mschira,

That should work very well :) In the end it comes down to the specific tools (FSL, AFNI, ANTS ...) if they support AMD specific instructions when they were compiled, but I haven't seen any issues on AMD Epyc or Ryzen CPUs. Agree that Intel is currently not competitive. Nvidia CUDA support is working in our internal tests, so the RTX should be fine as well. I would stay away still from AMD GPUs and ARM CPUs as the support for these is still lacking.

All the best
Steffen

**Comment by mschira** [comment]
Thanks Steffen,
Good to hear you haven't heard of any specific problems with AMD CPUs. I have heard some anecdotal type evidence of a friend that tried to run Neurodesk containers on his home gaming computer with an AMD CPU and Nvida GPU running Windows. He reported that he had no luck because "AMD apparently does virtualisation at bit different".  
So in other words vague on detail, but enough for me to check it out at the source, least that I end up with a new $5000 machine being a nightmare because we chose AMD for the extra 20-30% of performance.
Best
Mark

**Reply by stebo85** [reply]
Interesting to hear. If you can organise details about the amd virtualisation problem in windows we could have a look.
